packages/components/psammead-inline-link/src/index.stories.jsx

```diff
@@ -24,4 +55,98 @@ storiesOf('Components|InlineLink', module)
       </>
     ),
     { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'inline-link inside a regular paragraph',
+    ({ text, longText, script, service }) => (
+      <>
```

Comment:
We don't need to wrap this in a `Fragment` anymore since we're only retuning one component now.

The same applies to lines `75`, `88` and `101`.